c/tskit/core.c

```diff
@@ -794,8 +794,8 @@ tsk_search_sorted(const double *restrict array, tsk_size_t size, double value)
             upper = mid;
         }
     }
-    offset = (int64_t)(array[lower] < value);
-    return (tsk_size_t)(lower + offset);
+    offset = (int64_t) (array[lower] < value);
+    return (tsk_size_t) (lower + offset);
```

Comment:
I'm not sure why clang format is adding this space, but I'm fairly certain that my editor is picking up the clang-format config.